web/components/FcDrawerViewReportsAtLocation.vue

```diff
@@ -0,0 +1,402 @@
+<template>
+  <div class="fc-drawer-view-reports-at-location d-flex flex-column">
+    <v-progress-linear
+      v-if="loading"
+      indeterminate />
+    <template v-else>
+      <div>
+        <div class="align-center d-flex flex-grow-0 flex-shrink-0 px-3 pt-2">
+          <v-btn
+            icon
+            @click="actionNavigateBack">
+            <v-icon>mdi-chevron-left</v-icon>
+          </v-btn>
+          <h1 class="subtitle-1">Turning Movement Count</h1>
+          <div class="mx-1">&#x2022;</div>
+          <div>{{location.description}}</div>
+
+          <div
+            v-if="filterChips.length > 0">
```

Comment:
here we should take out all chips related to the `studyType` filter - maybe a computed property could help, similar to what we did with `filterParamsPaginated`?